app/src/androidTest/java/com/epfl/neighborfood/neighborfoodandroid/ui/activities/ProfileEditingActivityTest.java

```diff
@@ -27,53 +26,44 @@
 import android.provider.MediaStore;
 
 
-import androidx.lifecycle.Lifecycle;
+import androidx.test.core.app.ApplicationProvider;
 import androidx.test.espresso.intent.Intents;
 import androidx.test.ext.junit.rules.ActivityScenarioRule;
-import androidx.test.ext.junit.runners.AndroidJUnit4;
 import androidx.test.platform.app.InstrumentationRegistry;
 
+import com.epfl.neighborfood.neighborfoodandroid.NeighborFoodApplication;
 import com.epfl.neighborfood.neighborfoodandroid.R;
 import com.epfl.neighborfood.neighborfoodandroid.models.User;
+import com.epfl.neighborfood.neighborfoodandroid.models.UserTestImplementation;
 import com.epfl.neighborfood.neighborfoodandroid.repositories.AuthRepository;
-import com.epfl.neighborfood.neighborfoodandroid.repositories.AuthRepositoryTestImplementation;
-import com.epfl.neighborfood.neighborfoodandroid.ui.activities.MainActivity;
-import com.epfl.neighborfood.neighborfoodandroid.ui.activities.ProfileEditingActivity;
 
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
-import org.junit.rules.RuleChain;
-import org.junit.runner.RunWith;
 
-import javax.inject.Inject;
 
-import dagger.hilt.android.testing.HiltAndroidRule;
-import dagger.hilt.android.testing.HiltAndroidTest;
+//@RunWith(AndroidJUnit4.class)
 
-@RunWith(AndroidJUnit4.class)
-//@HiltAndroidTest
 public class ProfileEditingActivityTest {
     public static final String KEY_IMAGE_DATA = "data";
-
+    private AuthRepository authRepo;
     @Rule
     public ActivityScenarioRule<ProfileEditingActivity> testRule = new ActivityScenarioRule<>(ProfileEditingActivity.class);
-    //public HiltAndroidRule hiltRule = new HiltAndroidRule(this);
-    /*@Rule public RuleChain rule = RuleChain.outerRule(hiltRule)
-            .around(testRule);*/
-    /*@Inject
-    AuthRepository authRepository;
-*/
+
     @Before
     public void setUp() throws Exception {
+
         Intents.init();
-  //      hiltRule.inject();
+        NeighborFoodApplication app = ApplicationProvider.getApplicationContext();
+        authRepo = app.getAppContainer().getAuthRepo();
+        authRepo.logOut();
     }
+
     @Test
     public void buttonSaveTest(){
 
-        onView(withId(R.id.saveButton)).perform(click());
+        onView(withId(R.id.saveButton)).perform(scrollTo(),click());
         //assertTrue(testRule.getScenario().getState() == Lifecycle.State.DESTROYED);
```

Comment:
I see that some of the assertions are removed  from the tests that were failing on cirrus. It's good that we will have less problems with our build maybe we should rectify these tests later on.